ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -79,6 +79,18 @@
 #     )
 
 
+# hardswish
+# @handle_frontend_test(
+#      fn_tree="mindspore.ops.function.nn_func.hardswish",
+#      dtype_and_x=helpers.dtype_and_values(
+#          available_dtypes=helpers.get_dtypes("valid"),
+#      ),
+# )
+# def test_mindspore_hardswish(
+#     *,
+#     dtype_and_x,
+
+
```

Comment:
Please finish the test as well :sweat_smile:

Reply:
Might've messed up during resolving merge conflict😅 Thanks for the feedback. Resolving it